components/miro_custom_app/actions/create-sticky-note/create-sticky-note.mjs

```diff
@@ -0,0 +1,43 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  name: "Create Sticky Note",
+  version: "0.0.2",
+  key: "create-sticky-note",
+  description: "Creates a sticky note on a Miro board",
+  type: "action",
+  props: {
+    miro_custom_app: {
+      type: "app",
+      app: "miro_custom_app"
+    },
+    boardId: {
+      type: "string",
+      description: "Board ID",
+      optional: false,
+    },
```

Comment:
Here you can create an `async options()` prop to get a dropdown menu to select a `boardId` using the api.